mithril-common/src/crypto_helper/mod.rs

```diff
@@ -19,6 +20,7 @@ pub use era::{
 };
 pub use genesis::{ProtocolGenesisError, ProtocolGenesisSigner, ProtocolGenesisVerifier};
 pub use types::*;
+pub use verification_key::*;
```

Comment:
Idea: we could create a `types` module for all the wrapped types we will implement?

Reply:
Good idea! IMHO that should be in a separate PR.